cvat/apps/engine/rq.py

```diff
@@ -153,9 +157,13 @@ def for_job(cls, job: RQJob):
     def for_meta(cls, meta: dict[str, Any]):
         return cls(meta=meta)
 
-    def save(self) -> None:
+    def save(self, *, pipeline: Pipeline | None = None) -> None:
         assert isinstance(self._job, RQJob), "To save meta, rq job must be set"
-        self._job.save_meta()
+        # TODO: send PR to the upstream repo to support pipelines
+        # self._job.save_meta()
```

Comment:
Can we remove commented code?

Reply:
It's the previous "normal" implementation of this function, not just commented code